.gitignore

```diff
@@ -0,0 +1,5 @@
+# IntelliJ project files
```

Comment:
Technically speaking, also correct. The file can be deleted as such, since it really doesn't serve any purpose in the repo anyway. Tnx